templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,204 @@
+              name: name,
+              showCallbackID: showCallbackID,
+            });
+          };
+
+          Mozilla.UITour.hideMenu = function(name) {
+            _sendEvent('hideMenu', {
+              name: name
+            });
+          };
+
+          Mozilla.UITour.getConfiguration = function(configName, callback) {
+            _sendEvent('getConfiguration', {
+              callbackID: _waitForCallback(callback),
+              configuration: configName,
+            });
+          };
+
+          Mozilla.UITour.trackHelloStatus = function () {
+            Mozilla.UITour.getConfiguration('availableTargets', function(config) {
+              if (config.targets && config.targets.indexOf('loop') > -1) {
+                sendMetric('hello-eligible-view');
+              } else {
+                sendMetric('hello-ineligible-view');
+              }
+            });
+          };
+
+          function isTargetAvailable (target, callback) {
+            Mozilla.UITour.getConfiguration('availableTargets', function(config) {
+              callback(config.targets && config.targets.indexOf(target) > -1);
+            });
+          }
+
+          function bindHelloObserver () {
+            Mozilla.UITour.observe(function(e) {
+              switch (e) {
+                case 'Loop:ChatWindowOpened':
+                  sendMetric('hello-start-conversation');
+                  break;
+                case 'Loop:RoomURLCopied':
+                  sendMetric('hello-url-copied');
+                  break;
+                // bug 1118346
+                case 'Loop:RoomURLEmailed':
+                  sendMetric('hello-url-emailed');
+                  break;
+              }
+            });
+          }
+
+         function handleVisibilityChange () {
+            if (document.hidden) {
+              // hide Hello menu & stop observer when changing tabs or minimizing window
+              Mozilla.UITour.observe(null);
+            } else {
+              // listen for Hello menu/chat window events
+              bindHelloObserver();
+            }
+          }
+
+          function thumbsUp () {
+            bindHelloObserver();
+            Mozilla.UITour.showMenu ('loop');
+            // bug 1115227
+            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_snippet-{{ snippet_id }}');
+            // bug 1113896
+            document.addEventListener('click', function(e) {
+                Mozilla.UITour.hideMenu ('loop');
+            }, false);
+            document.addEventListener('visibilitychange', handleVisibilityChange);
+            sendMetric('hello-open');
+          };
+
+          var buttons = document.getElementsByClassName('launch-hello');
+          for (var i=0; i < buttons.length; i++) {
+            buttons[i].addEventListener('click', function(e) {
+              e.preventDefault();
+              isTargetAvailable('loop', function(ret){
+                if (ret === true) thumbsUp()
```

Comment:
This `if` statement could be formatted better according the the style guide to be more readable:

```js
if (ret === true) {
  thumbsUp()
} else {
  sendMetric('hello-ineligible-click');
  // ...
}
```